### PR TITLE
Update the misleading comment for cifar10.py's softmax_linear layer

### PR DESCRIPTION
**Pull request by a7744hsc**
A fix for the issue #5251, make the comment more meaningful.

### CODE REVIEW

**Comment by vrv** [comment]
Can you sign the CLA?  If not, can you close this PR?

**Comment by a7744hsc** [comment]
I signed it!

File: tensorflow/models/image/cifar10/cifar10.py [hunk 1]
```diff
@@ -256,7 +256,8 @@ def inference(images):
     local4 = tf.nn.relu(tf.matmul(local3, weights) + biases, name=scope.name)
     _activation_summary(local4)
 
-  # softmax, i.e. softmax(WX + b)
+  # linear layer(WX + b),
+  # no softmax ops here because tf.nn.sparse_softmax_cross_entropy_with_logits() only accept unscaled logits.
```
Comment (Tripper3) [inline tensorflow/models/image/cifar10/cifar10.py]: Cant seem to download anything

Comment (a7744hsc) [inline tensorflow/models/image/cifar10/cifar10.py]: sorry, but I can't understand your comment.

Comment (vrv) [inline tensorflow/models/image/cifar10/cifar10.py]: "We don't apply softmax here because tf.nn.sparse_softmax_cross_entropy_with_logits accepts the unscaled logits and performs the softmax internally for efficiency"

Also, please wrap lines to 80 columns.  Thanks!

**Comment by vrv** [comment]
(Made the change myself, thanks!)

Skipping tests since this is just a doc change.

**Comment by vrv** [comment]
@tensorflow-jenkins test this please (just for sanity check)

File: tensorflow/models/image/cifar10/cifar10.py (hunk 1 above)
Comment (a7744hsc) [inline tensorflow/models/image/cifar10/cifar10.py]: Thanks for your reminder, I'll be more careful.
